### PR TITLE
Possibility to run workflows manually

### PR DESCRIPTION
**Pull request by Hi-Fi**
## Description

Allows running just deployment step manually within workflows (e.g. update parameters or make redeploy for some reason)

## Context

[DEVOPS-256](https://helsinkisolutionoffice.atlassian.net/browse/DEVOPS-256)

### CODE REVIEW

**Comment by Hi-Fi** [comment]
Can't be tested before workflows end up to default branch.

File: .github/workflows/stop_review.yml [hunk 1]
```diff
@@ -2,6 +2,13 @@ name: Stop Review
 on:
   pull_request:
     types: [closed]
+  workflow_dispatch:
+    inputs:
+      head_ref:
+        description: "Branch where review environment was created from (as in GitHub). Has to give as branch is probably removed and not available at the list."
```
Comment (Jokinen) [inline .github/workflows/stop_review.yml:8]:
```suggestion
        description: "Branch where review environment was created from (as in GitHub). Has to be given as the branch is probably removed and no longer available in the branch list."
```

**Review by Jokinen — APPROVED** [review]
Looks good. 

I'm wondering how this works. As far as I understand this would allow us to more easily configure settings (after we change the repo into using runtime configurations).

Do I understand correctly that the intended use for updating for instance the production configs would be to open a new branch with the fixed configurations, and then manually running the production deployment job with that branch?

As I understand this change also allows PR envs to be stopped on demand in the case you do not need it, or in case one has failed to stop?

**Comment by Hi-Fi** [comment]
> Looks good.
> 
> I'm wondering how this works. As far as I understand this would allow us to more easily configure settings (after we change the repo into using runtime configurations).
> 
> Do I understand correctly that the intended use for updating for instance the production configs would be to open a new branch with the fixed configurations, and then manually running the production deployment job with that branch?
> 
> As I understand this change also allows PR envs to be stopped on demand in the case you do not need it, or in case one has failed to stop?
> 
> The browser test is still really finnicky with the first authentication routine even with more added wait time. It's unrelated with this PR.

I'll create demo app to test this PR, as it might have some glitches which can't be checked before things are in default branch.

Main idea is to allow running of stop-review if environment is not needed or it has failed remove for some reason. The build related triggers are bonus to be able to redeploy app without need to build every time (if changes are coming from secrets).

**Comment by terovirtanen** [comment]
Workflows have been removed, DevOps pipelines are used instead

**Pull request closed without merging**